src/libs/actions/Policy.js

```diff
@@ -236,22 +251,37 @@ function createAndGetPolicyList() {
  * @param {String} policyID
  */
 function loadFullPolicy(policyID) {
-    API.GetFullPolicy(policyID)
-        .then((data) => {
-            if (data.jsonCode !== 200) {
-                return;
-            }
-
-            const policy = lodashGet(data, 'policyList[0]', {});
-            if (!policy.id) {
-                return;
-            }
-
-            Onyx.set(`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`, {
-                ...allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`],
-                ...getSimplifiedPolicyObject(policy, true),
+    const key = `${ONYXKEYS.COLLECTION.POLICY}${policyID}`;
+    const getFullPolicy = () => {
+        API.GetFullPolicy(policyID)
+            .then((data) => {
+                if (data.jsonCode !== 200) {
+                    return;
+                }
+
+                const policy = lodashGet(data, 'policyList[0]', {});
+                if (!policy.id) {
+                    return;
+                }
+
+                const currentPolicy = _.clone(allPolicies[key]);
+                const simplifiedPolicyObject = getSimplifiedPolicyObject(policy, true);
+
+                const updatedPolicy = {
+                    ...currentPolicy,
+                    ...simplifiedPolicyObject,
+                    pendingInvitations: [],
```

Comment:
This should be added via `getSimplifiedPolicyObject(policy, true);` but not this function

Reply:
`getSimplifiedObject` is  to extract data from server. 
`pendingInvitataions` is local data and not stored in server. The logic is different.

Reply:
`getSimplifiedObject` also used by `getPolicyList`. If it is  only  used in `loadFullPolicy` then it is fine, but it is used in other place.